app/javascript/pages/browse/TileListItem.vue

```diff
@@ -87,7 +99,7 @@ export default {
   flex-wrap: wrap;
   padding: 0.2rem 0.3rem 0.3rem;
   &--withShadow {
-    box-shadow: 0px 0.1rem 0.1rem rgba(0, 0, 0, 0.2);
```

Comment:
maybe we remove this designation at this point, since we're not calling it and don't want shadow?